product_docs/docs/pgd/5/cli/command_ref/commit-scope/create.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: pgd commit-scope create
+navTitle: create
+---
+
+## Synopsis
+
+The `pgd commit-scope create` command is used to create a commit scope in the EDB Postgres Distributed cluster.
+
+## Syntax
+
+```shell
+pgd commit-scope create <COMMIT_SCOPE> [OPTIONS] <RULE_DEFINITION> [GROUP_NAME]
```

Comment:
The command syntax `Usage: pgd commit-scope <COMMIT_SCOPE> create [OPTIONS] <RULE_DEFINITION> [GROUP_NAME]` 
Please note `<COMMIT_SCOPE>` after `commit-scope` and before `create` subcommands